isovar/variant_read.py

```diff
@@ -148,8 +150,13 @@ def variant_read_from_single_read_at_locus(read, ref, alt):
                     read))
             return None
 
+    if sequence[read_pos_before + 1:read_pos_after] != alt:
```

Comment:
Hm, not particularly - but there still might be some edges cases here?

REF: TCGAGATGCA
R1:    TC----GATGCA
R2:    TC--------TGCA

Variant is GA>"" at POS=3 - does the second read agree with the variant? I think it's both a philosophical and implementation question.